### PR TITLE
Fix bug missing Security Group in Advanced Zone with SG enabled

### PR DESCRIPTION
**Pull request by utchoang**
@davidjumani @rhtyd cc @svenvogel 
* Advanced Zone with SG enabled:
image

* Networks step:
image

* Security Group step:
image

### CODE REVIEW

File: src/views/compute/DeployVM.vue [hunk 1]
```diff
@@ -774,6 +780,9 @@ export default {
     },
     networkName () {
       return this.$route.query.name || null
+    },
+    showSecurityGroupSection () {
+      return this.networks.length > 0 && this.zone.securitygroupsenabled && this.hypervisor !== 'KVM'
```
Comment (davidjumani) [inline src/views/compute/DeployVM.vue:785]: @utchoang Why is KVM excluded for SG Selection ?

Comment (utchoang) [reply src/views/compute/DeployVM.vue:785]: @davidjumani 
I refer to https://github.com/apache/cloudstack/blob/master/ui/scripts/instanceWizard.js#L177 then I think KVM does not support it.

Comment (davidjumani) [reply src/views/compute/DeployVM.vue:785]: Got it! But in https://github.com/apache/cloudstack/blob/master/ui/scripts/ui-custom/instanceWizard.js#L1570 it does support it along with multiple IPs

Comment (utchoang) [reply src/views/compute/DeployVM.vue:785]: @davidjumani I understood. I'll remove `this.hypervisor !== 'KVM'` condition. Thanks.

**Comment by davidjumani** [comment]
@blueorangutan package

**Pull request merged**